eng/tools/compress-coverage.ps1

```diff
@@ -0,0 +1,16 @@
+param (
```

Comment:
I don't see this getting called. If it's not called let's delete it .

Reply:
removed